lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -457,4 +457,9 @@ private function parseColumnCommentFromSQL($column, $sql)
 
         return false;
     }
+    
+    private function pregQuote($str)
```

Comment:
Missing type declaration in parameter and return

Reply:
I was actually talking about `private function pregQuote(string $str): string` not the docblock =)